static-instrumenter/test-app/src/main/java/io/opentelemetry/contrib/staticinstrumenter/test/HttpClientTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.staticinstrumenter.test;
+
+import java.io.IOException;
+import org.apache.http.Header;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+
+@SuppressWarnings("SystemOut")
+public final class HttpClientTest {
+
+  private HttpClientTest() {}
+
+  public static void main(String[] args) throws Exception {
+    System.setProperty("otel.traces.exporter", "logging");
+    makeCall();
+  }
+
+  private static void makeCall() throws IOException {
+    try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
```

Comment:
Not a big deal but would recommend using OkHttp instead if possible, it's our general "go-to" HTTP client, hence already being in dependency management

Reply:
Oh, next time I'll default to OkHttp then ;)